purchases/src/main/java/com/revenuecat/purchases/interfaces/SyncPurchasesCallback.kt

```diff
@@ -0,0 +1,8 @@
+package com.revenuecat.purchases.interfaces
+
+import com.revenuecat.purchases.PurchasesError
+
+interface SyncPurchasesCallback {
+    fun onSuccess()
```

Comment:
Shouldn't this return a customer info? It does return one in iOS

Reply:
Hmm you're right... Big oversight, thanks!